### ISSUE TITLE
new "theme" idea's

### ISSUE THREAD

**Issue by Switch1024** [doki-theme/doki-theme-vscode#97]
**Is your feature request related to a problem? Please describe.**
No, its just a idea for newer themes

**Describe the solution you'd like**
How about we add some more, ehm.... professional images :)

**Describe alternatives you've considered**
Ofcourse i could add my own "professional" images, but its nicer to see other people enjoy them too (that didn't sound good at all)

**Additional context**
A small list of persons i'd love to see in that "set":
- Mai
- ZeroTwo
- Yuri (DDLC)
- Monika (DDLC)
- Hinata (the older one ofcourse)
- Aqua
- Megumin?

**Comment by Unthrottled** [comment]
When you say `professional` are you referring to `cultured` images? 

As a `professional` otaku, these are the type of images I like

<img src="https://raw.githubusercontent.com/doki-theme/doki-master-theme/master/readmeAssets/best_girls.png" align="center" alt="best girls" />

As a _gentleman_ of `culture` these are the types of images I like.

[Cultured image](https://doki.assets.unthrottled.io/backgrounds/rias_onyx_spicy.png)

So are you meaning `cultured` when you say `professional`? The distinction is important :smile:

**Comment by Switch1024** [comment]
you know what i mean :)

**Comment by Unthrottled** [comment]
> you know what i mean :)

If I knew what you meant, I wouldn't have asked for clarification, friend. However, I am going to assume that you are wanting me to content/images that are more sexually charged/suggestive. 

>  its nicer to see other people enjoy them too (that didn't sound good at all)

I agree, that is a good cause. However, I've got 61 themes now. Each of them where tailored to each character. While I don't explicitly track usage of themes, I do know that none of the `spicy` secondary content shows up on my CDN's usage. That's a lot of work to make something that I don't really want, and the large majority of people probably won't use. 

That's why I've added the `Custom Assets` to a majority of my plugins. So that you & others can make their experience unique & special. It also means that I am probably not adding any secondary content to any themes going forward. 

Plus, I also don't want to dance around the [GitHub Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines#sexually-obscene-content) and others. Don't feel like stirring the pot if I don't need to. Maybe that's because I lack culture 🤷 

Anyways, thank you for your interest suggesting these additions! I see that you've got good taste based on your list.
